src/main/java/it/gov/pagopa/pu/debtpositions/service/massive/action/InsertionActionMassiveDebtPositionService.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.debtpositions.service.massive.action;
```

Comment:
there is already a `sync` package... let's use this name for the base package instead of massive, it's uggly, but we could think about it later
```suggestion
package it.gov.pagopa.pu.debtpositions.service.installmentsync.action;
```